languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -26937,6 +26920,216 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="“,">„Die Fenster sind geschlossen<marker>,“</marker> sagt sie.</example>
             </rule>
         </rulegroup>
+        <rulegroup id="FRAGEZEICHEN_NACH_DIREkTE_REDE" name="Fehlendes Fragezeichen am Abschluss einer direkten Rede">
```

Comment:
Keines der oben genannten Beispiele bringt einen Treffer nach den neuen Regeln. Mein Problem sind eher Satzfragmente wie diese:
- Ist eine Weile her.
- Gibt ordentlich Muskeln.
- Hat eine ziemlich wirre Geschichte erzählt.

Das sind alles Beispiele aus Dialogen aus dem Roman "Limit" von Schätzing (den habe ich ins tatoeba-Format konvertiert um einen Pool von Sätzen zu haben, die im Großen und Ganzen korrekt sein sollten).
Nun sind das zwar anschauliche umgangssprachliche Dialoge, die in einem Roman sicher ihre Berechtigung haben, aber dennoch grammatikalisch falsch sind und daher von einer Regel als Frage- bzw. Aufforderungen oder Ausrufe interpretiert werden. Aus dem Bauch heraus würde ich sagen, wenn man in einen grammatikalisch unvollständigen Satz schreibt, muss man davon ausgehen, dass eine Grammatikprüfung so etwas anmerkt. Was mich allerdings wieder zweifeln lässt, ist, dass so etwas regelmäßig in einem professionell lektorierten Roman vorkommt.

zu 'bitte': Das Beispiel "Könnten Sie bitte kommen?" trifft genau den Kern. Um solche Satzkonstruktionen geht es mir in erster Linie, weil die Interpretation von 'bitte' als Verb zu falschen Interpretationen in den Kommaregeln führt. Vielleicht muss man 'bitte' auch als adverbial einstufen und beide 'bitte' und 'danke' als Interjektionen (Für die Kommaregeln ist das übrigens technisch unerheblich).

Zur ID: Da sind ja gleich zwei Flüchtigkeitsfehler drin. Da sieht man mal wie dringend ich so eine Rechtschreib- und Grammatikprüfung brauche :). Das wird korrigiert.